docs/src/concepts.md

```diff
@@ -0,0 +1,116 @@
+# Concepts
+
+## Acquisition Tools for Electrophysiology
+
+### Neuropixels Probes
+Neuropixels probes were developed by a collaboration between HHMI Janelia, industry partners, and others (Jun et al., Nature 2017). Since their initial release in October 2018, 300 labs have ordered 1200 probes. Since the rollout of Neuropixels 2.0 in October 2020, IMEC has been shipping 100+ probes monthly (correspondence with Tim Harris).
```

Comment:
1. Please hard wrap
2. Please implement the convention present in electrode-localization for references

Reply:
Hopefully the wrap worked. I thought VSCode was doing that automatically. 

References will be updated in the next commit